make/common/db/registry.sql

```diff
@@ -221,9 +221,11 @@ UNIQUE(namespace)
 );
 
 create table properties (
+ id int NOT NULL AUTO_INCREMENT,
  k varchar(64) NOT NULL,
  v varchar(128) NOT NULL,
- primary key (k)
+ PRIMARY KEY(id),
+ UNIQUE (k)
```

Comment:
Please record the schema changes in tools/migration/changelog.md so that  the others can finish the upgrade script work.

Reply:
Done